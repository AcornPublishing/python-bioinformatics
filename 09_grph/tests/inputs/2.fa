>Rosalind_6002
GTTGAGGTTACGTGTATGTCGACGACGATATGCCCAGTAACTCGCACTATATGACACTGG
GGTAATTCCCCTGAGAGCAGCCCTGTGGCGCGACC
>Rosalind_5184
ATTAATTAGAGGTTCCCCGTTCCACATAAGTTTTGAGTGGACGGTCCACCGCCATTACGA
CGATGGACAGGGTGCCAATACCCCTAGGACGCT
>Rosalind_8145
TGCTAAAGAGGCATAGGCGCGATATGAGAGCAGCTTCAACTACAACGGGGTCCGCTGGAA
GATAGCGCACTCCGCCGGTCCCACCCTCTCACGGGTCTT
>Rosalind_3518
CACATCCGCAATGCGGTCGGTGGGGTCTTCTGTAGAAGGATCCGCATTGTAAGTGCAGAA
CAGGGAGTGAGCAATGGGCTAA
>Rosalind_1195
ATTACGAAGACCCTTGCTTTATTACTTTCGATGTAAAGCTTGGGCTACAAACACGATTAA
TTCAGCCCACATAGAGTGCAAGAATATCGTTAG
>Rosalind_5472
GTCGGAGAGGGCACGTCTCTTACAGCCGATAAAAAGGGTAGACTACTCGATATGGCACGG
TAGTTACCGCACATGCTGGCACTGTCACAAGAGGGTG
>Rosalind_2554
GATACGTAATACATAAGGGTGTGGCGTATGGACTGGACAGACTGTCTGAGAGGTCGTACA
GCACCCTTGTACCGAAGTAATATAACAGAACA
>Rosalind_7364
TTACCGACAACTGAGAGAGTGCGCCGAAAGCCCTTACTAGTTTGCTGATCTCGGCTCATT
AGGGAGCAAACGCTTTGGCCTGCT
>Rosalind_4566
AACGTTAGTATTCCTGCTGAGCGAACCGCCTAAAAGTAATAATACTCGATTGCAGGCGGC
TTGAGGTGAAACATATTCCGTACCTCTTGATATCTGGA
>Rosalind_1064
GAACTGCCGTCCAACCCAGTGAGTCGATTGACATGGGTACACTTCGTCGCTGCGAGCCGC
GAGGCCCATCTAGATGCAGTATCCCCAAATAA
>Rosalind_5193
GACGACATGTTTCCGATCCCGTAGGTGGGTCCTGAGATGGCTTGGAATTCCTATCCCTCG
ACCGTTGCCCGTGATGTGTGATTCCCTACAACGGCG
>Rosalind_3957
GAAAACCGACAATGGGGAACGCATAAAACTGAAGAGGCGTAGTTTTCGGCGCAGTAGCTG
TCTCACTACGTATAAAACATTCAAGGATGCAGG
>Rosalind_6515
AACGGAGCACCTGTACCGAAATTAGAGAGAGTGACATGCGACACAGCCAGTAAGTGCACT
TAGGGAGCGCGTCCCTTCGTC
>Rosalind_0726
TGACAGCGGATGTGTGATACATCACTACTTTCGGTCAAGTAACATCCGCACCACTGGACA
AGACAGGCCCCGCTGTGAATAGTATAC
>Rosalind_4374
GGGGCAGCTGGTATATAGGCCGACAAATAGCCCTACGGTAGTACACTAGGACAAAATTCG
GGCGAGTAGAACCACAGTCA
>Rosalind_0850
TAGTTCGATTTTGTCACTGTGTAGAGCTTAACTCGGTGCTTCTAGACTCGATACGTGTAG
ACAATTATTGACCTTCCTTTCAGCTGGCAA
>Rosalind_7493
ACGCGGAACAGGCTGCACCGACACCGCAGGATCTCTATCATAAATTGAAGACCAGGACAC
TACTCTTGGTTAACGTCGCCTACCCCGACGCGTCT
>Rosalind_3345
TGGGGTCCGGTCTATAAGCTGCGGTGTGGGGTTCGCTTCTTAAGGAGGCCTAGCGGACCA
TGTGTTGCAGCTTCACGTAACA
>Rosalind_9390
TCTTCGACGTGCATGTTCGTAGCGGTCGGCTGAGCATTTATCCCGGTGGTGCTACCTATG
GGACTGGACCTACGAGGGCATTTCCCCCACACTACCT
>Rosalind_3147
GTATGGCGTACGGTAGGCGCTAAAGTGAGGATCCTAAATAAGCGTAAAACTCTGAATGAC
AACCCCAGTTACTCCCAATG
>Rosalind_5867
GGGACCATAATAACTACGCGTGCGGCGGATGCCCAACTCGCCGCGGCTGGGGAATCGGTA
CAGGTATATGCGCCTAGTGAGCTCGATTTAACGG
>Rosalind_1502
ATGTGATCGCCTCAACGCCGATACCAACGTGTTTCGCCTTCTGAAGTGGAGTCGTGCTGG
TACATTTCGACATAATGGACATGAGAACCT
>Rosalind_9983
TCTCACGCTCATAAAAATCGCTCCTTCACCCCGGTAGGCCCCACGCTCCTGTTCTACGCG
CCTTCCCGACCATTGCGCACTGT
>Rosalind_4493
ATAGCATACGATAGCTCCTGTAGGCACGAAAGGACGTTGGAGTTGGTATCCATCAGGTAA
ATGTGAAGCTGTCTAAGCGCATCA
>Rosalind_9421
TCAGTCGCAGATGTCATCGGCAGACCGTTGCTAGTTGTTTCGAAGAAGCCCTTCGGTCCA
GATAATATGAGTCCAGGCGGGTCCATCGAGCTTGCTGTTG
>Rosalind_7052
ATCGTGATAAACGCAATGACTCACTACTGACCGGTTTTCGACAGGTAATGGCTGAGAAGG
GCCTGCTACGCCTACTCCCCCTGGTTACGCTACGCT
>Rosalind_2872
TGTCTATAATCAGACTAGCTGACGAAGCCTGTAGGACTATCCCATAAGACGAGGGAATTG
GCACAGAATAGTGGACAACCGGAAAG
>Rosalind_1084
TTAAAACCGTCTCAACACTGTTCCCCTATGAACACGCGCCTGTTCCATACTTTTTGAGCG
CCGAAATCCGAATCGGGCCTTCGGCCACGGC
>Rosalind_6708
GTGTCTCGCATATGGGTTGGTGTGATCGTACTACCTTGCACGTCTCAGTGAATCTAGCGG
GTATTTACTGATGCTGACCCGCC
>Rosalind_3470
GAGCTCAGTCGACCTTTCAAGGACGTTTAAGTTTCGCATGTCTCTTACAAAGGACACGGA
GTTGTGCTAGCCAACTATAT
>Rosalind_9945
CACGTAACGTAGCGGATCCTGTGGGTGTTGATTATGGCCAGCACCCAAGGTCGCCCTTAG
ACATCAACCCCTCTTGTCGCTTATCCGCAGGCGCTTAAAC
>Rosalind_8840
CATGGCTTGTTCTTTCCTCTTAACATATTACGAGATTCCCAGCCCCCCTAATGATTCCCA
CAGATCAATTAGGCAACAGCTAGGT
>Rosalind_0378
GTAACTTGGTTCGAAAGTGACCTGTTAAAGATTTCTATCCAATAAGCATTTCACACATGG
AGTACGTGGGTGCCTAGGATC
>Rosalind_0706
TGCCGGGACTACCCTACCGGGATAGGTTAGCCACCTCTATTACCATCTACGGGGCCGAGA
TTCTGCGTCTTTACTAAAACCAC
>Rosalind_4924
TTAGTTGCGTGGGACAGACACGGTGGTGTGGCTGTTATGGGTAGCACGCCCTAGGGTCAG
GCTATCAGTATAAAGCACAGGAGAACA
>Rosalind_1890
CAAAGGCGAGTCTAGCCATTACGCATTTATAACATTAGAAGCGCCCAAATTCCTTAACTG
CTAGCGATCCGCTCTTCATCACGC
>Rosalind_7871
GTGGAACAATTACAACAAATATAGCCATGATGTATTAACACGCGGTCCTCAACGACTTTC
TTCTCTGTGCAGCTCTGATCAAAC
>Rosalind_2095
CCCGCCTTCTCTATTTTCCACGTGCTGGCACTGTTCAAGTGCGGGGGTTCTCGTCGCATG
ACACCCTATGTCTTTCTTTCGAATTGCCGGCTTGGTGGAC
>Rosalind_2840
CCCTTGTTCCCACTACCTGCCATTTTGCGCCCCGTTTACCAGTGTGAGTGTGCACGGCCG
AACGGGATTCCGTGCGTCACATAGC
>Rosalind_6462
CTACTGTGTCGTAGATTCAAGCCCGTTTACGCTCGGCTTCCATACACTACTTGTCCTGCA
CTGACGTATATAAGATTCGCATATAGCCTCCTTTCCTCCC
>Rosalind_7148
AGCAATATACACACTTCGACAATATCATTTCACCCAGTCACCTGATTGACAACCATGAAT
GCGTGGGATTTGCGCAACGG
>Rosalind_2337
CAGCTAAATCTTGAGCACGTCAATTCATATGTGACGGCGGACAATCCTTTATGAAAGGCC
CGGGGAGACAGATTCACGCGACCTACC
>Rosalind_9122
GTTACAGTGGCTCTCGAGTCAACAGTAACAGTGTAGCCTCAATTATCACTTGGACTCGAG
TCCCGGTAAGACCCGGCCCTCTACCTTT
>Rosalind_0691
CCAAAGCTCCCACATTACCGCCAGTTAGATTGCAGTATGGCACACGTTCACTGAAAGTTT
GGCGTATTAATGTTAATGAGATACCGCGTGCGCGAG
>Rosalind_5119
CTATAAACGCACTTATTTCGAAATTGTTTCCAAACAAGCGATTCTCGGGCCATGGATCAC
GAACGCATCAACAAGACGGGCGAGTAAA
>Rosalind_6711
CCATTGAACATTGCAACCCCGGAGTAGCGCGAATTCAGGTAGTTACCACGACCGGGCATA
GCGATGCCGGAATTCCTGTAAAGCTCCGC
>Rosalind_2208
AATGTGCTAAATAACACAAATTCATAAACTGAGGTCTAGAACGAGTTTTGAGCGACAACC
TTGACGTTGCAAGTCGTTCTTCCCTTT
>Rosalind_9850
CTTGTGTACCTATGACTGTAACGGCGAGTCTTACCATCCCCGTCGCACACACCGTTAAAG
CAGATAACAAGTGACATCTCAGATAAGTGA
>Rosalind_0281
TATATTCCTCTAACATAACCTGTTTGTGCTTACATCATTGAGCGAAACCTTGCAGGTAGT
TGAAACGTGCCAGAAGAGCG
>Rosalind_9346
CATGTATCTATGAGGCTAACTCCCAAGCCGCGCGCATATGTTACGTAATTGGTCTGGAAG
ATGGGTTACAAGGGCGTGGCGTTTCCGGGACGAGGA
>Rosalind_7653
ATGGCGCTGCCCCCGCATATGTTGAGTCGGTTTTGGGGAGCCCTTCCGGAAAGCTCTGCG
CAGTTATTGTATAAAGCGACGGGCCGTTACATCGACGTTT
>Rosalind_3614
TCCAATTTAGGACCACTGCGTAGGGTAATCCGGCCGCACCGGGTCCCGAGTCCATATTCG
ATACTACTCTTTCGTTTTTCTCTATTC
>Rosalind_1280
CGGATACAGATACAAAACTCACAACTCGAGTAATTTCCCGGCGGTTGGTTACTCGCGGAA
CCGGCTATCTCTACGATACG
>Rosalind_4581
TAGTTACCTAGACCCGTTCCTTGGATGGAACCAGCATCAGCACGATAGTTGTGGTTACAT
ATTAGTGACCGTTAGCATAAAGTAGTGGGGC
>Rosalind_9149
ATTGCAGATACCTTGCTGTAGACGTCTCCCACCTTGCTGTCAGAGTTAACGCTCTAGTTA
TGACTAATTGGGTAATGCACGACG
>Rosalind_6802
CACCTTGACAATGGGACCTAGGTACCACCGCGGAGAGCACTACTAGTAATTCGTATGTCT
GCGGGGGTTCCTGCATAGGCGCCGCATCTT
>Rosalind_3888
GATTAACTTGTAAAAGGTCTAGAAGGAACAATAACCACAAGGCACTCGAATCGTCCAGTC
TCCCCCCGTAGGCATCCCCTATGTA
>Rosalind_5168
TCTTGCATCAAAGACTCATACATGGGTGATGCTCACCCACAATACTTATGAGCGCCGAAG
AAGAAGGGCGCTCCAGCAGA
>Rosalind_4536
TCGGACAGAGATATAAGCTATATAGCTTCAGGCGGTCCAACATTTGCCCGTTAATGACTG
CGTAGGCTGCGCTAACAAGCAACGCTTAGGGTTGCCTT
>Rosalind_4731
GCCGACTACCCGCGTTCCGCAACATAACCCGTCACCAGATGCGGACTGGAGTAAGGAATT
CCTGGATATGGAAACTTTTACTTCACAATTCCTGCCTTTG
>Rosalind_0505
GTGTTAAAGTCCACGAGTATTGCTTAGACAATATAGAGCGCATCAATAGAAAGCACAGTG
AAATCTCTGGACACGCAGCTTGCATGGAGTATGTTTGGT
>Rosalind_1144
GTTCCTGGCGATTTACGTACACGCATATAAAGTACGAAGCTACTGATGGCTGGAGCTTGC
TACCGCGACTGCACCAGAATTGACTCGGGACTGCGAAT
>Rosalind_7135
GGATCTATCTGAATTTCTGGGATCCGATATTTTCCGCGCGTGCAGGAACTTCTCTGTTAT
ACTCAGCTAGCGTGATCGTGATTCACGCATGAC
>Rosalind_2130
ACTGTTCTAGATCGGTACATGGCGGATCCCTGATGCTGCGACGTGCATAGCCGATCTACG
AATTTGAGAATTCTAGGCTTAAGAAGCATAGA
>Rosalind_0491
TAGCGGGGGGGCCGCTAACAGTTCCCTAAGGGGCACTAGGTGGGGCGAACTATCGTCCGC
TAGTGGGACCTCTTGTTGACAAGAA
>Rosalind_2666
TGGATTCGTGAGTTATCATATAAAGACCTGGGTACTAATACGGAGAGTGTAGTACAACTT
TGAGGTAACCATTACAGCGGATAGTAATACGTCAATGGCG
>Rosalind_8831
ATCGCTCTCGCTGAAACTTATAGTATTAACCAGTTACGGATGCCTAACGGAACCACTCGA
CTTCAGAGGGCGATCATACGTACCGCAAACAAGC
>Rosalind_5870
CGAATGCGCAAAGCGCGCCAACAATTTAGTATACGCGATAGAAGCAGGAATTATGATTGA
TGAAAATCGATCGTTATTGA
>Rosalind_6194
TACTCGGTCGGGTTCATCTCAGCTAGAGTTCAGTGTTTATCCCGGGTGCCGAATCACTGT
CCCCCTCCCACTACATTTTAGG
>Rosalind_0614
GGGCACACAGTCGCTCCGAGTACCATACACCAGAGACGCCCAAGAACCCGGGTTCCTAAT
ATGTTAGCGAGTTTGTTATGCACAGCTGATTGG
>Rosalind_9894
CAGCGTGTGGCGTTACGATCGTGTTCCTTTAGTGTTTAAAAATATACCTATAGCCACCTG
TGGGCCTGTTTCGCGTTTAGCATCGATACA
>Rosalind_8788
AAGGCGCCAGGGAAGCATGGTGACAAGCTAGGCCGTTTAGTGCGGATTCTGGAGTATTGA
TACGTACCGTTATGAACTGTATCC
>Rosalind_6084
CAATTTTGTCTCGGAGTAGGTTTGCATACCAATATGACACCTTCGGGTAAGGCCAGAACT
TTACAATGAATGGGCTGGACGACACAGTGCTCC
>Rosalind_0001
GACTTCGTGGCAAAAAAAATTGAGGCACAGCCATTCTACGCTAGGAAGACTCTCCACAAA
ATGTTGCCGTCTTGTATAATCGCATGTGCG
>Rosalind_5860
ACTAGAAGATCAAGTCTTGACTCTTCCTAAGTAGTTATTGTGGCTCCTTTATATTTATCA
GGAGGCGTGCTTTCAGATGGCGT
>Rosalind_3377
GATTTCCTATCCAATTATCTCTTCTAATTATTGGCTAGCTGACTGTTCTCGACCCCAAGC
GCGAGGCAGCGTCAGACTTCGCATACACAGACCGG
>Rosalind_3966
GGTGGACTCTTCGATTGTATCTTTGTACTAGCCTCCGCTTGTTTTAGTATGCAGGCATCT
CCTATCGGCTACCATTCTTGCGTAGGGTCTGTGCTGAG
>Rosalind_0914
CGATAATTATGTGCGTTCGTTCGAACAGCGAGCAGCGAGGCTTGAGTCCTCAGAATGCCT
TAAACAACACATCCCCACCGACAATCCCCTT
>Rosalind_3624
GCCAACACTCTTCGGTCGTACGCCCAGAAACACCCTGATAAGCAGCGTTCCCCCAATTAT
GGCCTGGGATAGCCCAGCCCCCCTTC
>Rosalind_1844
GAAATAACCCAACGAGCGTGCGCGTGAAACCCCAAAATGCCACGGCACAGACCTGTCTGG
AATACACACTACACTACGACGGG
>Rosalind_1836
TTCGTTGTGTGATGTATTGGCACGCCGGCAAAACACTTACATGCGTTTCTTGTTGCATTG
GGTGTGTCAATCCAAGTCAATATAGACGCCCCTCAAC
>Rosalind_6911
CTGACGGGCGCGAGTAAGTCTCCCGAACACAGCGGGCGGGTGCACTTGCCTTACTGTTTG
CTTTAACTTAGCATTACCTGAAGTGCTCGCTAAATC
>Rosalind_6988
CTGCTAGAATAGTTTGCAGGTTGGTCGATTCCAAAGTCCCCATTTCTCGGCCACAGCGGT
CCTTCACATAAATCAAGGTTGTTCGCTTGCG
>Rosalind_2900
GCTAGATATGACATACGATGTGATCGCGTACCCTCATATGTCTCCTTTTGGCACATGTCG
TGATGTCTCTCCCTGCAGTCGCTTCCATTC
>Rosalind_4067
GCCCCAAAACCTTGACGGCCACCTTCGGAATCGAGAACACTCATCGGCCGGCATTACATC
CGGATGCCAGCACCTCGTATCCGTTGT
>Rosalind_9726
ATGAATGTGGCACGATCTTTGGTGTCTAAGCAACCAAGCTATGTAGCGAGTCCCATCATA
ATGTGCTCAGGGGCCTACTCCAGGAACGTA
>Rosalind_0045
TGGACTTCTATAGTAATGAAGACCGCTCAGAAGAAAACCATGCTTCCAGTCTTATTACCA
GTTTCAAATGGGGACTCAATCGTGCGCGATGCGTGTATC
>Rosalind_5933
GTCCCTTCCCGTTCACTTGTAGACATTAAACTGAGTTGTTAAACTTGTTACGCGTACTCA
ATCTCGCAGTTTGGGTTGTAATATAAGCT
>Rosalind_7173
TGGGCCGTCTGGAGCGTTGATGTCGCCGACACCCTCGACAACGTTATCTTGGGTGCCAAA
CAACGCTCTAACCAACTGTCCCAG
>Rosalind_8309
CCCCGCGGAAGGGTCTATACATGGCTGATAAAGCCCAGGGCGTCCGATTACACCCGCACC
GGAAAATGTTCATGGGGTGTTGGCAAATCGTGCTCTGCA
>Rosalind_4304
ACTCTAGCCTAGACGCGTGTACATCTCACTCTGCAGGTTTGCGACGCCTGACCGAGGGTC
GGTAGTCATACCAGTTACCAA
>Rosalind_7751
GGTAGTGAGAGCTTATCGTGACTAAGTTTCACTCAGTGGAACTGAAACTGCACGGCAGTG
GCTTCCCGGTACACTCTCCCATAAGTTAGAAAAA
>Rosalind_0507
GGAATAGTTCGTGCTAACAAAGCGGCTAATGAGATTCAGGCCGGCAGCTCAGCGATACCC
CTGGATATATAATAAAACGAGGA
>Rosalind_3293
GGTGTGTTCACGTAGGAGGTGAAATGAAATAGGGTAGCCAGGCTCTGCGCGTCTCTAACG
TGCTCCATGAAGTCAGAAGCGACATCA
>Rosalind_7158
TGGTAACTTACTATAAAGTCTCGGGGAAGGCTATGACACATTCTGCTTAGGGCATTTGAG
TAAAGGCCGCTAGAGCCTCAGCTCGCATTACCA
>Rosalind_2943
GACGTGGCACTCGCCAAGACTGGTGCTAACGAAGTTGTAGAATTAGTGTGCCAGTCTAGT
CGGCGGTATGGCGGTTGTGTTATAACCGTTCC
>Rosalind_1517
AAAAAAGATGGCTTCATATCCACGTCTTAGGTCGGTTGATTTCGGCCGGACGCATAGATT
CGAACACTACAGTCTTGCGCGTCAACAGTTAATC
>Rosalind_7595
CAAGAGGGGGCTTCAAACTACAAAGCTTCTAACGACCCCGGCCAGCGTTGTTGTTTCCCG
CGTGACTCATGTAGGGGAAAGCCAATAGGGCCA
>Rosalind_1126
ATTAATCCAATTAACTTCAACCCCGCGTGGAGCGTAATTTTGCCACGTCGAAGATTGGAC
TTCAAGGTCACCCAGTATCATCATTCAC
>Rosalind_2576
GCTGGAACGCTCTCAAAATGTACACAGATCACGGGATCACAGGGGCACGTTTGAGCTGCC
GTCTAAAGCGCACGCCTTGCATACTTACTACGATGGATA
