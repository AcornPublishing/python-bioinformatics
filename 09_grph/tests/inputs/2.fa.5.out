Rosalind_0378 Rosalind_7135
Rosalind_0914 Rosalind_2840
Rosalind_1144 Rosalind_5870
Rosalind_2095 Rosalind_0045
Rosalind_2840 Rosalind_4493
Rosalind_2943 Rosalind_1144
Rosalind_3470 Rosalind_0281
Rosalind_4581 Rosalind_4374
Rosalind_5867 Rosalind_6515
Rosalind_7148 Rosalind_6515
Rosalind_7364 Rosalind_6988
Rosalind_7751 Rosalind_1517
Rosalind_9983 Rosalind_2130
